src/dc_receive_imf.rs

```diff
@@ -813,7 +813,7 @@ async fn add_parts(
             if let Some(chat_id) = chat_id {
                 if Blocked::Not != chat_id_blocked {
                     chat_id.unblock(context).await?;
-                    chat_id_blocked = Blocked::Not;
+                    // Not assigning Blocked::Not to avoid unused_assignments warning.
```

Comment:
```suggestion
                    // Not assigning `chat_id_blocked = Blocked::Not` to avoid unused_assignments warning.
```

I think I wouldn't understand the comment otherwise without the context what was there before.